chronicle/release/releasers/github/test-fixtures/create-v0.1.0-dev-repo.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+if [ -d "/path/to/dir" ]
+then
+    echo "fixture already exists!"
+    exit 0
+else
+    echo "creating fixture..."
+fi
+
+git init repos/v0.1.0-dev-repo
+
+pushd repos/v0.1.0-dev-repo
+
+git config --local user.email "nope@nope.com"
```

Comment:
Definitely ok with splitting it up. I don't think this is super urgent, since we're never going to actually email these folks, but I remember hearing that the owners of `test.com` or similar are very tired of getting bot spam from everyone's QA accounts, and `example.com` is the canonical valid domain name that no one uses.